### DISCUSSION TITLE
build error

### DISCUSSION THREAD

**Discussion by liuyangc3** [starcoinorg/starcoin#2429] [Q&A]
steps
```shell
./scripts/dev_setup.sh
source /root/.cargo/env
cargo build --all
```
outputs 
![image](https://user-images.githubusercontent.com/6705844/115174467-de323580-a0fb-11eb-953a-07cdec18c068.png)
...
![image](https://user-images.githubusercontent.com/6705844/115174483-e8543400-a0fb-11eb-92e3-4d1f4b0b73a6.png)

**Comment by jolestar** [comment]
内存不够，编译进程被操作系统 killed 了，增加虚拟内存。

**Reply by liuyangc3** [reply]
机器最小内存需要多少

**Reply by jolestar** [reply]
> 机器最小内存需要多少

这个没试过，你的配置是？

**Reply by liuyangc3** [reply]
2 vCPU 4 GiB

**Reply by jolestar** [reply]
加虚拟内存应该可以
